docs/src/examples/components/Menu/Variations/index.tsx

```diff
@@ -49,6 +49,26 @@ const Variations = () => (
       description="A vertical menu may take the size of its container."
       examplePath="components/Menu/Variations/MenuExampleVerticalFluid"
     />
+    <ComponentExample
+      title="Menu with Icons"
+      description="Menu items can contain icons."
+      examplePath="components/Menu/Variations/MenuExampleWithIcons"
+    />
+    <ComponentExample
+      title="Vertical Menu with Icons"
```

Comment:
Titles should reflect props when possible: 
"Vertical Icon" for `<Menu vertical icon />`